jupyter_rsession_proxy/__init__.py

```diff
@@ -29,6 +29,18 @@ def get_icon_path():
         os.path.dirname(os.path.abspath(__file__)), 'icons', 'rstudio.svg'
     )
 
+def rewrite_auth(response, request, orig_response, host, port, path):
```

Comment:
```suggestion
def rewrite_auth(response, request):
```

Only need to put in the signature the stuff you actually use!

Reply:
Good catch, thanks!